static/app/views/settings/projectMetrics/projectMetrics.tsx

```diff
@@ -151,21 +192,25 @@ function ProjectMetrics({project, location}: Props) {
 
 interface MetricsTableProps {
   isLoading: boolean;
-  metrics: MetricMeta[];
+  metrics: MetricWithCardinality[];
   project: Project;
   query: string;
 }
 
 function MetricsTable({metrics, isLoading, query, project}: MetricsTableProps) {
   const blockMetricMutation = useBlockMetric(project);
   const {hasAccess} = useAccess({access: ['project:write']});
+  const cardinalityLimit =
+    project.relayCustomMetricCardinalityLimit ?? DEFAULT_METRICS_CARDINALITY_LIMIT;
 
   return (
     <StyledPanelTable
       headers={[
         t('Metric'),
+        <Cell right key="cardinality">
+          {t('Cardinality')}
```

Comment:
Would be nice to have an arrow pointing down icon to indicate that the table is sorted by cardinality.